app/controllers/api/v1/exports_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class ExportsController < ApplicationController
+      include HasDoorkeeperAuthentication
+
+      def index
+        search_result = SearchResult.find(params[:search_result_id])
```

Comment:
As this is for rendering a search result, should the action name be `show` instead?
```suggestion
      def show
        search_result = SearchResult.find(params[:search_result_id])
```